test/middlewares/order-model.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { Response } from "express";
+import { orderModel } from "./../../src/middlewares/order-model";
+import { col } from "sequelize";
+import { SequelizeQueryParserRequestInterface } from "../../src/interfaces";
+import { SEQUELIZE_QUERY_PARSER_DATA_NOT_FOUND_ERROR } from "../../src/constants";
+
+const db = require("./../../example/db");
+describe("Order Middleware", () => {
+  let req: Partial<SequelizeQueryParserRequestInterface> & {
+    sequelizeQueryParser: any;
+    query: any;
+  };
+  let res: Partial<Response>;
+  let next: jest.Mock;
+
+  beforeEach(() => {
+    req = {
+      sequelizeQueryParser: {
+        model: db["User"],
+      },
+      query: {}, // Define query here
+    };
+    res = {
+      status: jest.fn().mockReturnThis(), // Ensure 'status' returns 'this' (res) for chaining
+      json: jest.fn(),
+    } as unknown as Response; // Explicitly define res as Response
+    next = jest.fn();
+  });
+
+  it("should set default order when req.query.order is undefined", async () => {
+    const middleware = orderModel();
+    await middleware(
```

Comment:
Remove async await since order middleware doesnt perform any async operation. Same goes to all test in this file